src/styles/blocks/container.scss

```diff
@@ -0,0 +1,47 @@
+.cotainer {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+
+  &__block {
+    height: $blockHeight;
+    min-width: $blockHeight;
+    text-align: center;
+    line-height: $blockHeight;
```

Comment:
You can use grid properties `justify-items` and `align-items` for centering content

Reply:
yep, use grid or flex for centering the number inside `&__block`